src/index.html

```diff
@@ -4,10 +4,327 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="icon" href="images/favicon.png">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+    <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+
+    <header class="header">
+      <div class="header__container">
+        <div class="header__top">
+          <div class="header__nav-icon" id="toggler"></div>
+
+          <nav class="nav" id="navigation">
+            <div class="nav__menu">
+
+              <ul class="nav__list">
+                <li class="nav__item">
+                  <a href="#" class="nav__link">Home</a>
+                </li>
+                <li class="nav__item">
+                  <a href="#aboutUs" class="nav__link">About Us</a>
+                </li>
+                <li class="nav__item">
+                  <a href="#gallery" class="nav__link">Gallery</a>
+                </li>
+                <li class="nav__item">
+                  <a href="#contacts" class="nav__link">Consultation</a>
+                </li>
+                <li class="nav__item">
+                  <a href="#testimonials" class="nav__link">Testimonials</a>
+                </li>
+                <li class="nav__item">
+                  <a href="#contacts" class="nav__link">Contacts</a>
+                </li>
+              </ul>
+
+              <a href="tel:+12345555555" class="nav__tel-wrap">
+                <p class="nav__tel">call to order</p>
+              </a>
+
+            </div>
+          </nav>
+
+          <a href="#">
+            <img
+              src="./images/logo.png"
+              alt="Miami Logo"
+              class="header__logo"
+            >
+          </a>
+
+          <a href="tel:+12345555555" class="header__tel-wrap">
+            <div class="header__tel"></div>
+          </a>
+        </div>
+
+        <div>
+          <div class="header__heading-line"></div>
+          <h1 class="header__heading">
+            Miami
+            <br>
+            Condo Kings
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <main class="main-content">
+
+      <a class="anchor" id="aboutUs"></a>
+      <section class="first-section main-content__section">
```

Comment:
```suggestion
      <section class="about-us main-content__section">
```
Let's give more meaningful names to your sections (about-us or about-us-section etc).
It's not a very good idea to name your sections (that have some meaningful content) with numbers